.github/workflows/ddg-release.yml

```diff
@@ -198,107 +189,54 @@ jobs:
                   path: autoconsent/
             - uses: ./autoconsent/.github/actions/setup-release-scripts
             # --- Action ---
-            - name: Checkout iOS
+            - name: Checkout Apple monorepo
               uses: actions/checkout@v3
               with:
-                  repository: duckduckgo/iOS
-                  path: ios/
+                  repository: duckduckgo/apple-monorepo-test # TODO: Update with final monorepo URL
```

Comment:
The final URL will be `apple-browsers`, but we're using a test repo for the time being. This needs to be replaced before merging.